### PR TITLE
CLOUDP-86202: OLM bundle changes for 0.5.0

### PR DESCRIPTION
**Pull request by antonlisovenko**
OLM Bundle changes:
* description
* maturity/channels

### CODE REVIEW

**Review by theburi — APPROVED** [review]

**Comment by antonlisovenko** [comment]
Some tests have fallen due to timeouts - this seems to be linked to the GCP capacity problems in cloud-qa.
Merging

File: docs/dev/olm.md [hunk 1]
```diff
@@ -37,9 +38,15 @@ The following happens "under the hood":
 
 As per https://sdk.operatorframework.io/docs/building-operators/golang/quickstart/:
 ```
-make bundle-build BUNDLE_IMG=antonlisovenko/test-bundle:v0.3.0
```
Comment (viv718918) [inline docs/dev/olm.md:40]:
```suggestion
make bundle-build BUNDLE_IMG=antonlisovenko/test-bundle:v0.3.0
```

File: docs/dev/olm.md [hunk 2]
```diff
@@ -37,9 +38,15 @@ The following happens "under the hood":
...
-make docker-push IMG=antonlisovenko/test-bundle:v0.3.0
```
Comment (viv718918) [inline docs/dev/olm.md:41]:
```suggestion
make docker-push IMG=antonlisovenko/test-bundle:v0.3.0
```

File: docs/dev/olm.md [hunk 3]
```diff
@@ -37,9 +38,15 @@ The following happens "under the hood":
...
-operator-sdk run bundle docker.io/antonlisovenko/test-bundle:v0.3.0
```
Comment (viv718918) [inline docs/dev/olm.md:42]:
```suggestion
operator-sdk run bundle docker.io/antonlisovenko/test-bundle:v0.3.0
```

File: Makefile [hunk 4]
```diff
@@ -22,7 +23,7 @@ endif
 # To re-generate a bundle for any other default channel without changing the default setup, you can:
 # - use the DEFAULT_CHANNEL as arg of the bundle target (e.g make bundle DEFAULT_CHANNEL=stable)
 # - use environment variables to overwrite this value (e.g export DEFAULT_CHANNEL="stable")
-DEFAULT_CHANNEL=alpha # TODO change to beta when we release Trial
```
Comment (viv718918) [inline Makefile:25]:
```suggestion
DEFAULT_CHANNEL=alpha # TODO change to beta when we release Trial
```

**Review by viv718918 — COMMENTED** [review]
改

**Pull request merged**